src/main.ts

```diff
@@ -1,8 +1,16 @@
 import * as core from '@actions/core';
+import * as Sentry from '@sentry/node';
+
 import {getCLI} from './cli';
 import * as options from './options';
 import * as process from 'process';
 
+Sentry.init({
+  dsn: 'https://156a0eddcf407521a39833be862fc0e6@o1.ingest.sentry.io/4506112609943552',
+  // Performance Monitoring
+  tracesSampleRate: 1.0,
+});
+
```

Comment:
Can we do this? A lot of repos use this action, wouldn't this collect data?
https://github.com/getsentry/action-release/network/dependents